### ISSUE TITLE
Using CaseInsensitiveClassMapBuilder for standard field mapping

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#105]
```
Hi,

I would be useful if you can use CaseInsensitiveClassMapBuilder not only for 
byDefault method but also for all other field mappings.
```

Original issue reported on code.google.com by `Mathias....@gmail.com` on 27 May 2013 at 12:12

**Comment by GoogleCodeExporter** [comment]
```
Please take a look at the latest master; an enhancement has been applied that 
should now allow for this...
```

Original comment by `matt.deb...@gmail.com` on 17 Jun 2013 at 4:05
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 1.4.3 release
```

Original comment by `matt.deb...@gmail.com` on 27 Jun 2013 at 2:25
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Great, thanks!
```

Original comment by `Mathias....@gmail.com` on 28 Jun 2013 at 9:14
